.github/workflows/build.yml

```
@@ -254,11 +254,9 @@ jobs:
           - debian12-systemd
           - debian13-systemd
           # # Only Debian-based platforms are currently supported.
```

Remove extra comment level:
```suggestion
          # Only Debian-based platforms are currently supported.
```